docs/csharp/codedoc.md

```diff
@@ -12,16 +12,464 @@ ms.devlang: dotnet
+    /// </summary>
+    public class SomeClass
+    {
+        /// <summary>
+        /// <c>DoStuff</c> does stuff on an instance of <c>SomeClass</c>
+        /// </summary>
+        public void DoStuff()
+        {
+
+        }
+    }
+
+<a name="code"></a>
+## &lt;code&gt;
+
+The &lt;code&gt; tag is used to indicate that multiple lines of text should be marked as code.
+See the [&lt;example&gt;](#example) tag for sample usage.
+
+<a name="para"></a>
+## &lt;para&gt;
+
+The &lt;para&gt; tag is for use inside a tag, such as &lt;summary&gt;, &lt;remarks&gt;, 
+or &lt;returns&gt;, and lets you add structure to the text.
+
+    /// <summary>
+    /// <para>This class does something</para>
+    /// </summary>
+    public class SomeClass
+    {
+        public void DoStuff()
+        {
+
+        }
+    }
+
+<a name="list"></a>
+## &lt;list&gt;
+
+The &lt;list&gt; tag lets you format documentation information as an ordered list, unordered list or table.
+
+    /// <summary>
+    /// This class does something
+    /// </summary>
+    public class SomeClass
+    {
+        /// <summary>
+        /// An example of an unordered list
+        /// <list type="bullet">
+        /// <item>
+        /// <term>Some term</term>
+        /// <description>Some description</description>
+        /// </item>
+        /// <item>
+        /// <term>Some term</term>
+        /// <description>Some description</description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        public void DoStuff(int a)
+        {
+
+        }
+
+        /// <summary>
+        /// An example of an ordered list
+        /// <list type="number">
+        /// <item>
+        /// <term>Some term</term>
+        /// <description>Some description</description>
+        /// </item>
+        /// <item>
+        /// <term>Some term</term>
+        /// <description>Some description</description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        public void DoMoreStuff(int a, string b)
+        {
+
+        }
+
+        /// <summary>
+        /// An example of a table
+        /// <list type="table">
+        /// <listheader>
+        /// <term>Table Heading Col 1</term>
+        /// <term>Table Heading Col 2</term>
+        /// <term>Table Heading Col 3</term>
+        /// </listheader>
+        /// <item>
+        /// <term>Col 1 Row 1</term>
+        /// <term>Col 2 Row 1</term>
+        /// <term>Col 3 Row 1</term>
+        /// </item>
+        /// <item>
+        /// <term>Col 1 Row 2</term>
+        /// <term>Col 2 Row 2</term>
+        /// <term>Col 3 Row 2</term>
+        /// </item>
+        /// </list>
+        /// </summary>
+        public void DoALotMoreStuff(params string[] args)
+        {
+
+        }
+    }
+
+<a name="see"></a>
+## &lt;see&gt;
+
+The &lt;see&gt; tag lets you specify a link from within text. Use the `cref` attribute to create internal hyperlinks to documentation pages for code elements.
```

Comment:
Should the syntax for `cref` be explained? I think especially generics and method overloads could use some explaination.